app/controllers/tenants_controller.rb

```diff
@@ -3,6 +3,7 @@ class TenantsController < ApplicationController
   def index
     if params[:query].present?
       @tenants = Tenant.search_by_first_name_and_last_name_and_address(params[:query])
+      redirect_to tenants_path
```

Comment:
This will cause the query to not work anymore because it searches (if a query is present), then gets redirected back to the base page without the `query` in `params`